components/docs-chef-io/content/automate/ha_aws_deployment_prerequisites.md

```diff
@@ -83,8 +83,8 @@ Current Automate HA integrates with the following non-Chef tools:
 
 {{< note >}}
 
--   Use a [Hardware Calculator](/calculator/automate_ha_hardware_calculator.xlsx) to check how much hardware you will need for your use case.
--   Make sure the hardware requirement in not lesser than the recommended [Minimum Hardware Requirement](/automate/ha_aws_deployment_prerequisites/#minimum-hardware-requirement)
+- Please refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.
```

Comment:
Replace it with: 

- Refer to [Performance Benchmarks](/automate/ha_performance_benchmarks) for more details on the hardware requirements.

Reply:
done.